scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -359,6 +359,10 @@ def run_cmake(self, args="", filter_stages=[]):
         cmake_opts = ['-DBOARD={}'.format(self.platform.name)]
         cmake_args.extend(cmake_opts)
 
+        for this_snippet in self.instance.testsuite.required_snippets:
```

Comment:
why snippet needs to be added in cmake?

Reply:
Snippets are supplied at configuration time to cmake

Reply:
They're documented on https://docs.zephyrproject.org/latest/build/snippets/index.html with examples of how to use them